crates/biome_service/src/configuration/parse/json/mod.rs

```diff
@@ -1,14 +1,6 @@
 //! This module is responsible to parse the configuration from a JSON format
 //!
 
-mod configuration;
-mod css;
-mod files;
-mod formatter;
-mod javascript;
-mod json_impl;
 mod linter;
-mod organize_imports;
-mod overrides;
```

Comment:
I think if the `rules` submodule below gets cleaned up, which requires letting the generated rule structs also tap into the macro, it might make sense to just get rid of the `configuration::parse` module altogether. The few remaining manual deserializers could easily be implemented along the structs they implement.